### ISSUE TITLE
missing code in diverse_depth_model.py

### ISSUE THREAD

**Issue by RamsleyJunting** [YvanYin/DiverseDepth#4]
Hello, it seems that the code related to the loss(Ranking loss/WCEL loss/SSIL loss/MSGL_Loss/YouTube3D_Ranking_Loss) defined in ModelLoss is missing, could you provide them?

**Comment by YvanYin** [comment]
Hi,
We have not released the training codes right now.
